cli/lib/kontena/cli/certificate/show_command.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../services/services_helper'
+
+module Kontena::Cli::Certificate
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    parameter "SUBJECT", "Certificate subject"
+
+    requires_current_master
+    requires_current_master_token
+    requires_current_grid
+
+    def execute
+      certificate = client.get("certificates/#{current_grid}/#{self.subject}")
+      puts YAML.dump(certificate)
```

Comment:
It would be nice if this also supported some option like `kontena certificate show --cert test.188.226.131.23.nip.io | openssl x509 -noout -text`, equivalent to `kontena vault read --value LE_CERTIFICATE_test_188_226_131_23_nip_io_CERTIFICATE | openssl x509 -noout -text`:
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            03:e9:1d:72:49:86:e0:a3:7f:21:73:be:d0:0c:a7:48:a8:b2
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, O=Let's Encrypt, CN=Let's Encrypt Authority X3
    ...
```

May also want to be able to export the private key too?

Reply:
Can be added, IMO better to do in separate PR